### ISSUE TITLE
Tooltip styling doesn't appear

### ISSUE THREAD

**Issue by hlmr-erik** [open-formulieren/open-forms-sdk#494]
Having pulled the latest Docker image, I still don't see the styling. Perhaps there is no styling set by default? I couldn't find the relevant tokens in the token editor as well.

Hovering to show the tooltip does indeed cause a CSP-error in the console
`[Content-Security-Policy: The page's settings observed the loading of a resource at inline ("style-src"). A CSP report is being sent.](index.js:148 [Report Only] Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-ver4aEqWkp4pos9Mtcw7kw=='". Either the 'unsafe-inline' keyword, a hash ('sha256-4PbkY7AMmcyJNRDb0oCWDI3MntDOeDCgwed+y+9Fy0I='), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present.)`

Tooltip text:
![image](https://github.com/open-formulieren/open-forms-sdk/assets/111565013/baf73e99-c06b-45ea-b1a1-2628584ee36d)

Trying to find it in the token editor:
![image](https://github.com/open-formulieren/open-forms-sdk/assets/111565013/be698a2d-ef7f-48f6-b453-9ab11443fdc7)

EDIT: manually setting
`    "tooltip": {
      "arrow": {
        "color": {"value": "#ae9e49"},
        "height": {"value": "16px"},
        "width": {"value": "8px"}
      },
      "background-color": {"value": "#fff3cd"},
      "border-color": {"value": "{of.tooltip.arrow.color}"},
      "border-width": {"value": "1px"},
      "box-shadow": {
        "value": "1px 0 0 #e6e6e6, -1px 0 0 #e6e6e6, 0 1px 0 #e6e6e6, 0 -1px 0 #e6e6e6, 0 3px 13px rgba(0, 0, 0, 0.08)"
      },
      "font-size": {"value": "1rem"},
      "offset": {"value": "10px"},
      "padding-block-end": {"value": "6px"},
      "padding-block-start": {"value": "6px"},
      "padding-inline-end": {"value": "12px"},
      "padding-inline-start": {"value": "12px"}
    },
    "tooltip-icon": {
      "color": {"value": "{of.color.fg-muted}"}
    }`
as design tokens didn't change the appearance.

Test form: [tooltip-test.zip](https://github.com/open-formulieren/open-forms-sdk/files/12028849/tooltip-test.zip)

**Comment by hlmr-erik** [comment]
It appears I mistook what happened here. The Docker image of the fix hasn't been pushed? https://github.com/open-formulieren/open-forms-sdk/actions/runs/5530361998/jobs/10089669476 So I guess its not integrated into OpenForms proper yet. Closing for now

**Comment by hlmr-erik** [comment]
It appears I mistook what happened here. The Docker image of the fix hasn't been pushed? https://github.com/open-formulieren/open-forms-sdk/actions/runs/5530361998/jobs/10089669476 So I guess its not integrated into OpenForms proper yet. Closing for now

**Comment by hlmr-erik** [comment]
Still doesn't appear in the lastest docker image

**Comment by sergei-maertens** [comment]
I can't seem to reproduce this on our own test environment which was deployed with latest a couple minutes ago:

![image](https://github.com/open-formulieren/open-forms-sdk/assets/5518550/a1a5c2b0-4ac7-487c-987b-563cf04c7ac5)

**Comment by hlmr-erik** [comment]
Its finally showing now. Not sure if the image was updated since this morning, or if the command 'docker pull openformulieren/open-forms' fixed it.

This morning I tried by these inscructions: https://open-forms.readthedocs.io/en/latest/installation/docker_compose.html (download said zip-file, and run 'docker compose pull' in the extracted directory) and apparently that wasn't sufficient (or the image was updated since).
